panel/viewable.py

```diff
@@ -244,7 +244,7 @@ def servable(self, title=None):
         return self
 
     def show(self, port=0, websocket_origin=None, threaded=False,
-             title=None, verbose=True, **kwargs):
+             title=None, verbose=True, show=False, **kwargs):
```

Comment:
I don't see the `show` argument documented in the `show()` method docstring, which seems important given how unintuitive `show(show=False)` is. :-)

Reply:
Added the docstring and changed the default to the previous behaviour.

Reply:
I copied the docstring from `server.py`. Should kwarg show be popup or open?

Reply:
Oops; responded in different thread, proposing `open` or `launch` or `browse`.  Probably should let @philippjfr decide.

Reply:
No strong preference but I vote for `open` and I'd probably only rename it here.

Reply:
`show(show=True)` does look somewhat absurd (but I really want you to show...). I think the votes are in.

Reply:
Or even more strange to request `show(show=False)`

Reply:
I will push `open` but feel free to revert (`open` has the disadvantage of being a python command as well).